src/server/game/Handlers/MiscHandler.cpp

```diff
@@ -1203,7 +1203,8 @@ void WorldSession::HandleInspectOpcode(WorldPacket& recvData)
     WorldPacket data(SMSG_INSPECT_TALENT, guid_size+4+talent_points);
     data << player->GetPackGUID();
 
-    if (sWorld->getBoolConfig(CONFIG_TALENTS_INSPECTING) || _player->IsGameMaster())
+    // config value can be 0, 1 or INT_MAX - +1 guarantees that we end up with 0-3 (2-3 valid)
```

Comment:
`getIntConfig` returns `uint32`, `-1` -> `INT_MAX`, unsigned overflow is well-defined.